source/Score.js

```diff
@@ -0,0 +1,60 @@
+import Reference from "./Reference.js"
+
+
+export default class Score {
+    constructor(playerNumber) {
+        this.count = 0
+        this.playerNumber = (playerNumber === undefined) ? game.playerCount : playerNumber
+        this.domElement = document.createElement("div")
+        this.domElement.className += " player-score"
+        this.domElement.style.position = "absolute"
+        this.domElement.style.textAlign = "center"
```

Comment:
Unlike the color, which is dynamic for each `Junkership`, these CSS properties will always be constant, so why not put them in the CSS instead of appending them in the JS?